app/controllers/idv/document_capture_controller.rb

```diff
@@ -109,5 +111,25 @@ def handle_stored_result
         failure(I18n.t('doc_auth.errors.general.network_error'), extra)
       end
     end
+
+    def allow_direct_ipp?
+      # not allowed when no step param and action:show(get request)
+      return false if params[:step].blank? || params[:action].to_s != 'show' ||
+                      idv_session.flow_path == 'hybrid'
+      # Only allow direct access to document capture if IPP available
+      return false unless IdentityConfig.store.in_person_doc_auth_button_enabled &&
+                          Idv::InPersonConfig.enabled_for_issuer?(decorated_sp_session.sp_issuer)
```

Comment:
I chatted with @JackRyan1989 a bit about this, and filed LG-12768 for us to update our code to use `Idv::InPersonConfig.enabled_for_issuer?`. Don't hold off until we do it, though, just noting it for your awareness.

Reply:
@n1zyy , i used this condition based on like in the frontend
https://github.com/18F/identity-idp/blob/0d1343d2586e414cd6be1fafbe15cf3d8b618133/app/views/idv/shared/_document_capture.html.erb#L33-L33

Here I did not add `in_person_proofing_opt_in_enabled` because regardless of the `how to verify` opt-in screen enabled or not, the handoff page ipp option is not affected.

Reply:
@n1zyy is there existing documentation about `IdentityConfig.store.in_person_doc_auth_button_enabled` and `Idv::InPersonConfig`?  I want to make sure this information is accessible outside of this github conversation.

Reply:
@eileen-nava I don't think there is (yet). :( For want of a better solution, I left comments here as little breadcrumbs. I'm still not quite sure what the best way to capture this information is. Do you have a better sense than I do of where we could capture this knowledge?

Reply:
@n1zyy Since the knowledge that needs to be documented is Team Joy domain, I will leave the decision up to you of where to document it.